packages/terra-section-header/docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Terra Section Header
+
+Section Header presentational component that provides an onClick callback and animateable status icon.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-section-header`
+
+## Usage
+
+```jsx
+import React from 'react';
+import SectionHeader from 'terra-section-header';
+
+<SectionHeader
+  title="Section Header Example"
+  isAnimated
+  onClick={() => {}}
```

Comment:
Would it be valuable to actually provide an example of an onClick callback here?

Reply:
I used an alert call in Dialog to demo the close button. Something like that would probably be fine.

Reply:
Done 58a80795867b02ec7067de6d972b46cc2f78a633